time time
int64 id
---
geometry_msgs/Point[] traj